src/fetch-info.js

```diff
@@ -57,7 +57,7 @@ const specrefStatusMapping = {
 
 const fetchQueue = new ThrottledQueue(2);
 
-async function useLastInfoForDiscontinuedSpecs(specs) {
+function useLastInfoForDiscontinuedSpecs(specs) {
```

Comment:
That's fine but I thought I'd note that I think about it differently: this is one of the step functions, some of the step functions are asynchronous, I'd like all of them to be asynchronous for consistency, as such I want them to return a `Promise` which the `async` keyword guarantees.